tests/test_metrics.py

```diff
@@ -8,253 +8,343 @@
 
 
 class TestMetrics(TestDvc):
+
     def setUp(self):
         super(TestMetrics, self).setUp()
-        self.dvc.scm.commit('init')
+        self.dvc.scm.commit("init")
```

Comment:
Why this change in tests? ' to "?

Reply:
Will revert to original. I generally use black for formatting, and it automatically converts ' to "

Reply:
@tdeboissiere confirm this @efiop, btw. It looks like we are migrating to black, so may be should keep it your way.

Reply:
@shcheklein We've discussed it with @tdeboissiere on discord :slightly_smiling_face:  Let's merge https://github.com/iterative/dvc/pull/1547 first and then rebase this PR on top of it, so we do everything properly :slightly_smiling_face:

Reply:
👍